### ISSUE TITLE
minor display issue in rendered docs

### ISSUE THREAD

**Issue by ngam** [google/jax#10633]
Please:

- [x] Check for duplicate requests.
- [x] Describe your goal, and if possible provide a code snippet with a motivating example.

Split the `print(...)` statements into different cells in the notebooks so that they're better rendered in .md. If not split, they are usually rendered on separate cells. For example, the [following cell](https://jax.readthedocs.io/en/latest/notebooks/autodiff_cookbook.html#checking-against-numerical-differences)

```python
# Set a step size for finite differences calculations
eps = 1e-4

# Check b_grad with scalar finite differences
b_grad_numerical = (loss(W, b + eps / 2.) - loss(W, b - eps / 2.)) / eps
print('b_grad_numerical', b_grad_numerical)
print('b_grad_autodiff', grad(loss, 1)(W, b))

# Check W_grad with finite differences in a random direction
key, subkey = random.split(key)
vec = random.normal(subkey, W.shape)
unitvec = vec / jnp.sqrt(jnp.vdot(vec, vec))
W_grad_numerical = (loss(W + eps / 2. * unitvec, b) - loss(W - eps / 2. * unitvec, b)) / eps
print('W_dirderiv_numerical', W_grad_numerical)
print('W_dirderiv_autodiff', jnp.vdot(grad(loss)(W, b), unitvec))
```

gets rendered like

```
b_grad_numerical
```

```
 -0.29325485
b_grad_autodiff -0.29227245
```

```
W_dirderiv_numerical -0.2002716
W_dirderiv_autodiff -0.19909117
```

It is cumbersome to separate the print calls, but without a better fix in the underlying rendering, this is likely the best solution for now...

**Comment by ngam** [comment]
I first noticed this issue when I was trying to figure out [this part of the docs](https://jax.readthedocs.io/en/latest/notebooks/autodiff_cookbook.html#hessian-vector-products-using-both-forward-and-reverse-mode):

```python
# reverse-over-reverse, only works for single arguments
def hvp_revrev(f, primals, tangents):
  x, = primals
  v, = tangents
  return grad(lambda x: jnp.vdot(grad(f)(x), v))(x)


print("Forward over reverse")
%timeit -n10 -r3 hvp(f, (X,), (V,))
print("Reverse over forward")
%timeit -n10 -r3 hvp_revfwd(f, (X,), (V,))
print("Reverse over reverse")
%timeit -n10 -r3 hvp_revrev(f, (X,), (V,))

print("Naive full Hessian materialization")
%timeit -n10 -r3 jnp.tensordot(hessian(f)(X), V, 2)
```


```
Forward over reverse
4.29 ms ± 153 µs per loop (mean ± std. dev. of 3 runs, 10 loops each)
Reverse over forward
```

```
8.47 ms ± 4.63 ms per loop (mean ± std. dev. of 3 runs, 10 loops each)
Reverse over reverse
```

```
12.5 ms ± 7.41 ms per loop (mean ± std. dev. of 3 runs, 10 loops each)
Naive full Hessian materialization
```

```
47.2 ms ± 546 µs per loop (mean ± std. dev. of 3 runs, 10 loops each)
```

**Comment by ngam** [comment]
beautifully addressed by @ahtik in #10785, closing :)
